src/components/engagement/dto/status.enum.ts

```diff
@@ -26,6 +26,29 @@ export enum EngagementStatus {
   Rejected = 'Rejected',
 }
 
+export const TerminalEngagementStatuses: EngagementStatus[] = [
+  EngagementStatus.Terminated,
+  EngagementStatus.Completed,
+  EngagementStatus.Converted,
+  EngagementStatus.Unapproved,
+  EngagementStatus.Transferred,
+  EngagementStatus.NotRenewed,
+  EngagementStatus.Rejected,
+  EngagementStatus.DidNotDevelop,
+];
+
+export const NonTerminalEngagementStatuses: EngagementStatus[] = [
+  EngagementStatus.InDevelopment,
+  EngagementStatus.Active,
+  EngagementStatus.DiscussingTermination,
+  EngagementStatus.DiscussingReactivation,
+  EngagementStatus.DiscussingChangeToPlan,
+  EngagementStatus.DiscussingSuspension,
+  EngagementStatus.FinalizingCompletion,
+  EngagementStatus.ActiveChangedPlan,
+  EngagementStatus.Suspended,
+];
```

Comment:
If we are actually trying to partition the statuses into terminal/non-terminal, it would be better to define this as a map.
```ts
const EngagementStatusTerminalMap: Record<EngagementStatus, boolean> = {
  [EngagementStatus.InDevelopment]: false,
   ...
};
```
This will ensure that if/when statuses are added or removed that those changes are reflected here as well.

This mapping could be converted into two lists for consumption like they are now.